src/conversions/chrono.rs

```diff
@@ -445,7 +451,16 @@ impl<'py, Tz: TimeZone> IntoPyObject<'py> for &DateTime<Tz> {
     type Error = PyErr;
 
     fn into_pyobject(self, py: Python<'py>) -> Result<Self::Output, Self::Error> {
-        let tz = self.offset().fix().into_pyobject(py)?;
+        let tz = self
+            .timezone()
+            .into_pyobject(py)
+            .map(BoundObject::into_bound)
+            .map(Bound::into_any)
+            .map_err(Into::into)?;
```

Comment:
```suggestion
            .into_bound_py_any()?;
```
You could simplify this a bit with `IntoPyObjectExt`